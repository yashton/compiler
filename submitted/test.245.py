False
    False
    False
    True

False
    