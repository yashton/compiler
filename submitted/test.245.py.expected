(KEYWORD False)
(NEWLINE)
(INDENT)
(KEYWORD False)
(NEWLINE)
(KEYWORD False)
(NEWLINE)
(KEYWORD True)
(NEWLINE)
(DEDENT)
(KEYWORD False)
(NEWLINE)
(INDENT)
(DEDENT)
(ENDMARKER)
